algorithms/1-intro/intro_algo_answers.txt

```diff
@@ -0,0 +1,56 @@
+1.
+  a.)
+  DEF MAKE_AN_OMELETTE
+    HEAT pan on stove
+    CRACK eggs into mixing bowl
+    ADD milk TO bowl
+    REPEAT
+      whisk eggs
+    UNTIL eggs bubbly
+    ADD butter TO pan
+    IF butter completely melted
+      ADD mixed eggs to pan
+      REPEAT
+        tilt and turn pan
+      UNTIL pan covered entirely
+    END IF
+    IF mixed eggs not runny
+      ADD filling
+      FOLD egg
+    END IF
+    SET COUNTER to 0
+    WHILE COUNTER less than 31
+      COUNTER + 1
+    END WHILE
```

Comment:
Are we waiting 30 cycles? A more appropriate mechanism would be something similar to:

    SET SECONDS WAITED to 0
    WHILE SECONDS WAITED is less than 31
        WAIT one second
        INCREMENT SECONDS WAITED by 1
    ENDWHILE